packages/page-claims/src/Statement.tsx

```diff
@@ -0,0 +1,48 @@
+// Copyright 2017-2020 @polkadot/app-claims authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { StatementKind } from '@polkadot/types/interfaces';
+
+import React from 'react';
+import ReactMd from 'react-markdown';
+import styled from 'styled-components';
+
+import { getStatementMd, getStatementUrl } from './util';
+import { useTranslation } from './translate';
+
+export interface Props{
+  className?: string;
+  kind: StatementKind;
+}
+
+function Statement ({ className, kind }: Props): React.ReactElement<Props> | null {
+  const { t } = useTranslation();
+  const statementUrl = getStatementUrl(kind);
+
+  return (
+    <div className={className}>
+      {t('Please read the following statement carefully, it is also available at ')}
+      <a href={statementUrl}>{statementUrl}</a>
```

Comment:
```suggestion
      <a href={statementUrl} target="_blank">{statementUrl}</a>
```

so that they won't need to choose account & paste eth address when coming back to apps